tools/build/scripts/extract-api-summary.js

```diff
@@ -33,8 +33,10 @@ if (undefined !== argv.gatherFullReport)
   shouldGenerateFullReport = true;
 
 // create output file
-const trimmedApiSignature = (argv.apiSignature.split('.'))[0];
-const sigFileName = path.basename(path.normalize(trimmedApiSignature));
+const apiSignatureNormalized = path.normalize(argv.apiSignature);
+const apiSignatureFileName = apiSignatureNormalized.split('\\').slice(-1)[0];
+const trimmedApiSignatureFileName = (apiSignatureFileName.split('.'))[0];
+const sigFileName = path.basename(path.normalize(trimmedApiSignatureFileName));
```

Comment:
I don't think the file name needs to be normalized, especially since it's taken from an already normalized path.

Reply:
Removed